extensions/content-manager/admin/src/components/WysiwygWithErrors/index.js

```diff
@@ -15,19 +20,27 @@ const WysiwygWithErrors = ({
   onChange,
   value,
 }) => {
+  const user = useUser();
+
+  // We can distinguish admins by the role: "admin::roles.create"
+  const isAdmin =
+    user.userPermissions.find(
+      (x) => x.action === "admin::roles.create"
```

Comment:
This could use naming like (userPermission), since x, y or so lose the context easily.